src/components/SlateEditor/PlainTextEditor.tsx

```diff
@@ -42,10 +42,10 @@ const PlainTextEditor = ({
   const { status, setStatus } = useFormikContext<ArticleFormType>();
 
   useEffect(() => {
-    if (status === 'revertVersion') {
+    if (status?.status === 'revertVersion') {
       ReactEditor.deselect(editor);
       editor.children = value;
-      setStatus(undefined);
+      setStatus((prevStatus: any) => ({ ...prevStatus, status: undefined }));
```

Comment:
Går det an å type status?

Reply:
Skal prøve. Formik sin tanke er at den kan være hva som helst, men siden vi bruker dette som objekt kan jeg type det og legge typen i `interfaces.ts` og bruke typen der det brukes, slik at vi har hvertfall *litt* kontroll på hva status er for noe :+1: